sql/ddl_internal.sql

```diff
@@ -92,15 +92,16 @@ BEGIN
                                                 NULL,
                                                 chunk_time_interval,
                                                 NULL,
-                                                FALSE);
+                                                FALSE,
+                                                hide_interval_warnings=>TRUE);
```

Comment:
Why do we turn off warnings here?

Reply:
Because it is already checked for previously in the public API call. If we don't then two warnings are printed, the second containing a confusing field name (`interval` instead of `chunk_time_interval`, which is what the public API param is called).